lib/capybara_screenshot_idobata/rspec.rb

```diff
@@ -9,12 +9,10 @@
 
 CapybaraScreenshotIdobata.configure do |config|
   config.message_formatter = proc {|file, line, context|
-    example = context.respond_to?(:example) ? context.example : RSpec.current_example # For RSpec 2/3 conpatiblity
```

Comment:
I think it would be better to support RSpec 2 for `capybara_screenshot_idobata` users.
Are there any reasons to have to drop it?